tsconfig.base.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "baseUrl": "./",
     "module": "es2020",
+    "skipLibCheck": true,
```

Comment:
Was this definition added automatically?

Reply:
I added this definition because of a type error in @turf/turf version 6.
therefore, i added this variable, that ignores type errors in our node_modules

Reply:
Ok, this is new, I'm not sure how this will affect us, but we'll see.

Please answer all my other comments too.